src/Utility/RouteGuessingModelResolver.php

```diff
@@ -54,6 +52,6 @@ public function resolveModelClass(Route $route): string
      */
     final public function namespaceModel($model_class)
     {
-        return sprintf('%s%s', $this->getAppNamespace(), $model_class);
+        return sprintf('%s%s', App::getNamespace(), $model_class);
```

Comment:
`AppNamespaceDetectorTrait` was deprecated after Laravel 5.8 and rolled into the `Application`.